python/tests/test_concurrency.py

```
@@ -0,0 +1,44 @@
+def test_returns_value():
+    """Positional args are forwarded and the return value comes back."""
+    assert asyncio.run(sync_to_thread(lambda a, b: a + b, 2, 3)) == 5
+
+
+def test_propagates_exception():
+    """Exceptions raised in the worker propagate to the caller."""
+
+    def boom():
+        raise ValueError("nope")
+
+    with pytest.raises(ValueError):
```

Ruff (0.15.21)

[warning] `pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)